src/Microsoft.ML.SamplesUtils/SamplesDatasetUtils.cs

```diff
@@ -271,6 +271,58 @@ public static  IEnumerable<BinaryLabelFloatFeatureVectorSample> GenerateBinaryLa
             return data;
         }
 
+        public class FfmExample
+        {
+            public bool Label;
+
+            [VectorType(_simpleBinaryClassSampleFeatureLength)]
```

Comment:
>VectorType [](start = 13, length = 10)

am curious - is this attribute required ?

Reply:
It is required.

---
In reply to: [249994977](https://github.com/dotnet/machinelearning/pull/2205#discussion_r249994977) [](ancestors = 249994977)